src/interfaces/IOracleFactory.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.25;
+
+interface ILPOracleFactory {
```

Comment:
We may not need this... what do you think @lumoswiz

Reply:
Yeah, probably don't require it. We can see how it goes and remove later.

Reply:
It was necessary when I had four different factories for the bench marking utility function.